pkg/model/error.go

```diff
@@ -0,0 +1,5 @@
+package model
+
+type Error struct {
```

Comment:
maybe in the future we should also save a html error code here. For this we have to update the docs.